docs/tutorials/pydy_for_autolev_users.rst

```diff
@@ -0,0 +1,672 @@
+
+Rough Autolev-PyDy Equivalents
+----------------------------------
+
+The tables below give rough equivalents for some common Autolev
+expressions. **These are not exact equivalents**, but rather should be
+taken as hints to get you going in the right direction. For more detail
+read the built-in documentation on `SymPy vectors <http://docs.sympy.org/latest/modules/physics/vector/index.html>`__
+, SymPy mechanics and
+PyDy.
+
+**THIS IS AN EVOLVING DOCUMENT. If you find an error, or can fill in an
+empty box, please fix it! If there's something you'd like to see added,
+just add it.**
+
+In the tables below, it is assumed that you have executed the following
+commands in Python:
+::
+
+	from sympy.physics.mechanics import *
+	from sympy.physics.vector import *
+	from sympy import *
+	from pydy import *
+
+Mathematical Equivalents
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++-----------------------+-----------------------+-----------------------+
+| **Autolev**           | **PyDy**              | **Notes**             |
++=======================+=======================+=======================+
+||                      ||                      ||                      |
+| Constants A, B        | a, b = symbols(‘a b’) | Note that the names   |
+|                       |                       | of the symbols can be |
+|                       |                       | different from the    |
+|                       |                       | names of the          |
+|                       |                       | variables they are    |
+|                       |                       | assigned to. We can   |
+|                       |                       | define a, b =         |
+|                       |                       | symbols(‘b a’) but it |
+|                       |                       | is good practice to   |
+|                       |                       | follow the            |
+|                       |                       | convention.           |
++-----------------------+-----------------------+-----------------------+
+||                      ||                      ||                      |
+| Constants C+          | c = symbols(‘c’,      | Refer to SymPy        |
+|                       | nonnegative = True)   | assumptions for more  |
+|                       |                       | information.          |
+|                       |                       | `modules/core         |
+|                       |                       | <http://docs.sy       |
+|                       |                       | mpy.org/latest/module |
+|                       |                       | s/core.html>`__       |
++-----------------------+-----------------------+-----------------------+
+||                      ||                      ||                      |
+| Constants D-          | d = symbols(‘d’,      |                       |
+|                       | nonpositive = True)   |                       |
```

Comment:
Why use nonpositive versus positive? I forget whether there is a good reason to do one or the other.